README.md

```diff
@@ -172,155 +214,11 @@ Based on the size and strucutre of the dataset provided + these parameter settin
 | `do_not_balance_hdf5` | bool | False | Include flag for True, no arguments | Include this flag if you DO NOT want to balance HDF5 files, this is not recommended unless the you are dealing with a huge amount of data (many terra bytes), or do not want shuffling between splits. |
 | `keep_split_jsonls` | bool | False | Include flag for True, no arguments | If you DO NOT want to delete split jsonls files that are in text format in the `output_path/splits` directory include this flag. The only reason you would include this flag is if you want to see what text is in each HDF5, meaning that splits/train_1_of_X.jsonl is the jsonl text version of train_1_of_X.hdf5. Including this flag will increase the storage space of your dataset by more than two times. |
 | `num_workers` | int | False | 0 <= `num_workers`<= # of available CPUs | The number of CPU workers to run tokenization with, if the previous run failed due to OOM, you need to decrease this number. |
```

Comment:
should we place this in a dropdown? It takes huge amount of space

Reply:
Yeah makes sense, I defer. 👍 

Maybe we can ask a UI person from the company to see if there is a better way to display this table without the awkward scrolling?